web/src/js/components/FlowTable/FlowColumns.tsx

```diff
@@ -291,6 +292,13 @@ export const quickactions: FlowColumn = ({ flow }) => {
     );
 };
 
+export const comment: FlowColumn = ({ flow }) => {
+    const text = getComment(flow);
+    return <td className="col-comment">{text ? text : "..."}</td>;
```

Comment:
"..." sounds more like "too long to display" than no comment. Do we just want to leave it empty maybe?

Reply:
yes makes sense